arrows/core/video_input_image_list.cxx

```diff
@@ -162,10 +162,13 @@ ::set_configuration( vital::config_block_sptr in_config )
     "image_reader", config, d->m_image_reader );
 
   // Check capabilities of image reader
-  auto const& reader_capabilities =
-    d->m_image_reader->get_implementation_capabilities();
-  set_capability( HAS_FRAME_TIME,
-                  reader_capabilities.capability( image_io::HAS_TIME ) );
+  if( d->m_image_reader != nullptr )
```

Comment:
Personally, I tend to write null checks as `if( ptr )`, but I don't think we have any official guidance either way, and tastes vary.

Reply:
I'm trained by python, and don't remember its applicability in C++, where objects may overload that boolean cast, so I tend to default to being explicit about checks like this.